docker/polygon_zkevm/polygon_zkevm.sh

```diff
@@ -25,8 +27,8 @@ down() {
 
 configure_repo() {
     repo="https://github.com/0xPolygonHermez/zkevm-bridge-service.git"
-    commit_id="09e0247b335a19a5e16945e8e1e805e410e41ca3"
-    git init ${POLYGON_ZKEVM}
+    commit_id="59c1ba75fa19cf1156762a8c5ffa7f8fd272c60a"
+    git clone --no-checkout ${repo} ${POLYGON_ZKEVM}
```

Comment:
why we changed the initialization here

Reply:
@GabrielBuragev do you know? I haven't actually touched that.

Reply:
Probably a mistake while merging the two versions of the script on my end. Can you try with the old approach?

Reply:
ok, I've changed it back in the new pr https://github.com/beamer-bridge/beamer/pull/1977